.travis.yml

```diff
@@ -109,8 +109,3 @@ matrix:
     - rvm: 2.3.0
       env: RAILS_VERSION=master
   fast_finish: true
-
-branches:
-  only:
-    - master
-    - /^\d+-\d+-maintenance$/
```

Comment:
I know. I’m just trying to get a green build. I just added a "WIP" to the PR title.